src/styles/DropDownMenu.ts

```diff
@@ -42,11 +44,11 @@ export const DropDownItemRaw = {
       '&:hover': {
         backgroundColor: Colors.SecondaryHover,
       },
-      '&:active': {
+      '&.Mui-selected': {
```

Comment:
The background color is a bit strange when using the keyboard: it's sometimes grey and sometimes blue. Is there some adjustment to do?
(I'm fine with keeping it for a later PR too).

Reply:
Yes, I need to clean up these hover states and their borders. I think the native focus colors are also clashing. I'll create a separate PR for this.

Reply:
yeah, I remember I did some quick and dirty patch just to get some focus styles, but that this wasn't perfect.